examples/orders_create_and_download_multiple_orders.py

```diff
@@ -36,11 +36,18 @@
                      [-91.198465, 42.893071]]]
 }
 
-iowa_images = ['20200925_161029_69_2223', '20200925_161027_48_2223']
 iowa_order = planet.order_request.build_request(
-    'iowa_order',
-    [planet.order_request.product(iowa_images, 'analytic_udm2', 'PSScene')],
-    tools=[planet.order_request.clip_tool(iowa_aoi)])
+    name='iowa_order',
+    products=[
+        planet.order_request.product(
+            item_ids=['20200925_161029_69_2223', '20200925_161027_48_2223'],
+            product_bundle='analytic_udm2',
+            item_type='PSScene')
+    ],
```

Comment:
Using product()'s keyword arguments makes the calls more understandable.

Reply:
This makes it much more readable, thank you!